src/components/navbar/user-actions-menu.cjsx

```diff
@@ -75,9 +76,11 @@ UserActionsMenu = React.createClass
 
 
   render: ->
+    course = CourseStore.get(@props.courseId)
+
     <BS.NavDropdown
       eventKey={1}
-      className='-hamburger-menu'
+      className={classnames('-hamburger-menu', 'is-concept-coach': course?.is_concept_coach)}
```

Comment:
Ooh, cool! I think this could be used to show the student/teacher that they are in a concept coach (vs tutor) course which would be useful for pilot teachers that used CC last year but are trying Tutor this year